css/style.css

```diff
@@ -44,42 +36,31 @@ div.autocomplete-deluxe-container input.autocomplete-deluxe-form {
   color: #003CC5;
 }
 
-.content-header .is-horizontal-enabled:not(.is-horizontal) .tabs__link.js-tabs-link {
-  color: #545560;
-}
-
 .field--name-field-content .layer-wrapper .field-multiple-table {
   margin-bottom: 16px;
 }
 
 .paragraphs-features__add-in-between__wrapper {
-  padding-bottom: 4px;
+  padding-bottom: 7px;
 }
 
-.field--type-social-media .form-type--checkbox .form-item__label {
+.field--type-social-media .form-type--checkbox .form-item__label{
```

Comment:
Please make sure there is a space before the bracket

Reply:
resolved